google/cloud/spanner/internal/connection_impl.cc

```diff
@@ -1139,6 +1139,14 @@ StatusOr<spanner::CommitResult> ConnectionImpl::CommitImpl(
   request.set_return_commit_stats(params.options.return_stats());
   request.mutable_request_options()->set_priority(
       ProtoRequestPriority(params.options.request_priority()));
+  if (static_cast<Options>(params.options)
+          .has<spanner::MaxCommitDelayOption>()) {
+    *request.mutable_max_commit_delay() =
+        google::protobuf::util::TimeUtil::MillisecondsToDuration(
+            static_cast<Options>(params.options)
+                .get<spanner::MaxCommitDelayOption>()
+                .count());
+  }
```

Comment:
Use the new getter directly on `params.options`, rather than casting to `Options`.

Reply:
Done.